### PR TITLE
Kadane's Algorithm and Doubly Linked List Menu Driven Program C++ Implementation 

### PR DESCRIPTION
**Pull request by goyalh4164**
#51 #95 
Kadane's Algorithm is used to find the maximum contiguous subarray sum in O(N) Time complexity and O(1) Space complexity 
Here is the view of Output Screen of Kadane Algorithm

 
image (75)

Doubly Linked List Menu Driven Program :
 It Includes the following functions
1) Insert at Head
2) Insert at End
3) Insert at Position
4) Delete at Head
5) Delete at End
6) Delete at Position
7) Printing the Linked List
8) Reverse The linked list

Here is the view of Output Screen of Doubly Linked List
image (74)

Hope You Find it useful!!

### CODE REVIEW

**Review by Kumar-laxmi — COMMENTED** [review]
Please update the **README.md**

**Comment by goyalh4164** [comment]
@Kumar-laxmi README.md Updated Succesfully of both the issues.

**Comment by Kumar-laxmi** [comment]
@goyalh4164 pls update the repository README.md

File: C++/DoublyLinkedList/DLL.cpp [hunk 1]
```diff
@@ -0,0 +1,324 @@
+// C++ program for the all Doubly Linked List Operation
```
Comment (Kumar-laxmi) [inline C++/DoublyLinkedList/DLL.cpp:1]: Change file name to **DoublyLinkedList.cpp**

File: C++/Kadane-Algorithm/Kadane.cpp [hunk 2]
```diff
@@ -0,0 +1,60 @@
+#include <bits/stdc++.h>
```
Comment (Kumar-laxmi) [inline C++/Kadane-Algorithm/Kadane.cpp:1]: Please put this program in **Sub-Array** folder

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]

File: C++/DoublyLinkedList/DLL.cpp (hunk 1 above)
Comment (goyalh4164) [reply C++/DoublyLinkedList/DLL.cpp:1]: ok

File: C++/Kadane-Algorithm/Kadane.cpp (hunk 2 above)
Comment (goyalh4164) [reply C++/Kadane-Algorithm/Kadane.cpp:1]: ok

**Comment by goyalh4164** [comment]
@Kumar-laxmi Sir Everything is updated

**Comment by Kumar-laxmi** [comment]
@goyalh4164 Please resolve the **Merge Conflict**

**Comment by goyalh4164** [comment]
@Kumar-laxmi  All Merge Conflict Solved

**Comment by Kumar-laxmi** [comment]
> @Kumar-laxmi All Merge Conflict Solved

Still the **MERGE CONFLICT** is there

**Comment by goyalh4164** [comment]
@Kumar-laxmi All merge conflict are removed

**Review by Sanjanabharadwaj25 — COMMENTED** [review]
@goyalh4164 Please add readme for Kadane's algorithm in Sub-Arrays folder.

**Review by 072arushi — APPROVED** [review]
Approved

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Please don't update the main **README.md** for this PR, some changes are to be done.

You can remove the updation of **README.md** from this PR

**Review by Arun9739 — APPROVED** [review]
Approved !

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Please don't update the repo's README.md

**Comment by goyalh4164** [comment]
Ok I haven't updated the readme

File: README.md [hunk 3]
```diff
@@ -74,11 +74,6 @@
 | 37.  | Morris Traversal | [C](https://github.com/Kumar-laxmi/Algorithms/blob/main/C/Trees/morris_transversal.c), [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Trees/MorrisTraversal.cpp) | Trees |
 | 38.  | Red-Black Tree | [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Trees/red_black.cpp), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Trees/red_black.py) | Trees |
 
-
-
-
- </div>
```
Comment (Sanjanabharadwaj25) [inline README.md:80]: @goyalh4164 Don't update README.md. Remove it from your PR.

**Review by Sanjanabharadwaj25 — CHANGES_REQUESTED** [review]
@goyalh4164 README.md is changed in your PR. Don't include it in your PR.

**Comment by goyalh4164** [comment]
@Sanjanabharadwaj25 Everything is resolved

**Review by Sanjanabharadwaj25 — APPROVED** [review]

**Review by Kumar-laxmi — APPROVED** [review]

**Pull request merged**